app/bundles/CampaignBundle/Form/Type/CampaignType.php

```diff
@@ -157,8 +157,8 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
     /**
      * @return string
      */
-    public function getName()
+    public function getBlockPrefix()
     {
-        return 'campaign';
+        return self::class;
```

Comment:
This alias must be replaced with FQCN in the `CampaignModel`. The `getBlockPrefix()` method. The `'campaign'` string is used in a JS file but not as an indentifier, so the `getBlockPrefix()` method can be removed here as well.